atlasdb-config/src/main/java/com/palantir/atlasdb/factory/Leaders.java

```diff
@@ -132,10 +132,10 @@ public static LocalPaxosServices createInstrumentedLocalServices(
                 leadershipObserver,
                 ImmutableList.of());
 
-        PaxosAcceptor ourAcceptor = AtlasDbMetrics.instrument(metricsManager.getRegistry(),
+        PaxosAcceptor ourAcceptor = AtlasDbMetrics.instrumentTimed(metricsManager.getRegistry(),
                 PaxosAcceptor.class,
```

Comment:
I won't check them all, but just be weary of those pesky autodelegate implementations...

Reply:
Yep. None of the 6 classes here are annotated with `@AutoDelegate`